docker-compose.yml

```
@@ -0,0 +1,13 @@
+services:
+  postgres:
+    container_name: feedback-local-postgres
+    image: postgres:17
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
+    ports:
+      - 5432:5432
+volumes:
+  postgres-data:
```

[dust] If I could make a suggestion:

```suggestion
  postgres-data:
    name: feedback-data
    driver: local
    driver_opts:
      type: none
      o: bind
      device: ${PWD}/db

```

This allows you to continue to have a named volume but exposes the files in case for any reason you want to change config options or anything else, or easily nuke an existing database. If you take this approach, you'd just want to gitignore that folder.

---

This helps to prevent having to teach how to delete a named volume's data in case you need to reset the database.

---

Ah cool! We tried this out but decided to revert back to the original config because we got errors we had trouble resolving. But will definitely keep in mind if we run into the situation where we have to change config options on the db!!